tests/core_tests/test_context.py

```diff
@@ -118,6 +118,8 @@ def test_equal(self):
         context_2 = context.Context(self.tk, **kws2)
         self.assertTrue(context_1 == context_2)
         self.assertFalse(context_1 != context_2)
+        # Assert that hashing function treats these as equal
+        self.assertTrue(hash(context_1) == hash(context_2))
```

Comment:
Could you change `assertTrue/assertFalse` to `assertEqual/assertNotEqual` everywhere @pscadding ?

Reply:
So I'm pretty sure that if I switch to using those methods then the following lines should be condensed down to:
```
        self.assertTrue(context_1 == context_2)
        self.assertFalse(context_1 != context_2)
>
        self.assertEqual(context_1, context_2)
```

Reply:
OK I've updated all of those, please check they make sense. I did spot something I didn't understand, but it might be correct.